components/kyma-environment-broker/internal/broker/instance_update.go

```diff
@@ -105,6 +105,22 @@ func (b *UpdateEndpoint) Update(_ context.Context, instanceID string, details do
 		logger.Errorf("cannot fetch provisioning lastProvisioningOperation for instance with ID: %s : %s", instance.InstanceID, err.Error())
 		return domain.UpdateServiceSpec{}, errors.New("unable to process the update")
 	}
+	if lastProvisioningOperation.State == domain.Failed {
+		return domain.UpdateServiceSpec{}, apiresponses.NewFailureResponse(errors.New("Unable to process an update of a failed instance"), http.StatusUnprocessableEntity, "awer")
```

Comment:
nit: not sure if `awer` is intentional logging message, I wouldn't know what that means but seems to be unique in KEB :)

Reply:
it put it by accident, fixed